pkg/api/api.go

```diff
@@ -42,6 +43,18 @@ func NewAPI(o Options) (chi.Router, error) {
 		log:     &logger,
 	}
 
+	cors := cors.New(cors.Options{
+		// AllowedOrigins is used instead of AllowOriginFunc as it's faster and we don't need
+		// to allow credentials, so an asterisk works.
+		AllowOriginFunc:  func(r *http.Request, origin string) bool { return true },
+		AllowedMethods:   []string{"GET", "POST", "PUT", "DELETE", "OPTIONS"},
```

Comment:
Do we need anything other than POST?

Reply:
Not yet, though even if they're unused I'm assuming that we're going to add more capabilities to manipulate the dev server from a UI so it's not harmful :)